Source/Core/Common/GL/GLInterface/WGL.cpp

```diff
@@ -228,17 +228,26 @@ bool cInterfaceWGL::Create(void* window_handle, bool core)
       PFD_TYPE_RGBA,                  // Request An RGBA Format
       32,                             // Select Our Color Depth
       0,
-      0, 0, 0, 0, 0,   // Color Bits Ignored
-      0,               // 8bit Alpha Buffer
-      0,               // Shift Bit Ignored
-      0,               // No Accumulation Buffer
-      0, 0, 0, 0,      // Accumulation Bits Ignored
+      0,
```

Comment:
This probably calls for a "clang-format off" comment. We have those in other places (like the OpCode tables of Jit/DSP), don't know the exact syntax from the top of my head tho.